lib/modules/platform/gitea/index.ts

```diff
@@ -173,6 +175,47 @@ async function lookupLabelByName(name: string): Promise<number | null> {
   return labelList.find((l) => l.name === name)?.id;
 }
 
+function getRepoUrl(
+  repo: helper.Repo,
+  gitUrl: GitUrlOption | undefined
+): string {
+  if (gitUrl === 'ssh') {
+    if (!repo.ssh_url) {
+      throw new Error(CONFIG_GIT_URL_UNAVAILABLE);
+    }
+    logger.debug({ url: repo.ssh_url }, `using ssh URL`);
+    return repo.ssh_url;
+  }
+
+  // Find options for current host and determine Git endpoint
+  const opts = hostRules.find({
+    hostType: PlatformId.Gitea,
+    url: defaults.endpoint,
+  });
+
+  if (gitUrl === 'endpoint' || repo.clone_url === null) {
+    if (repo.clone_url === null) {
+      logger.debug('no clone_url found. Falling back to old behaviour.');
```

Comment:
```suggestion
      logger.debug('No clone_url found. Falling back to old behaviour.');
```

Is this the start of the debug message? If yes, then capitalize the first letter. 😉